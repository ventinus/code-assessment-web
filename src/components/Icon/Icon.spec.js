import React from 'react'
import renderer from 'react-test-renderer'
import { shallow } from 'enzyme'
import * as icons from '.'

const components = Object.values(icons)

describe('Icon component', () => {
  components.forEach(Icon => {
    it(`renders ${Icon.name} correctly`, () => {
      const tree = renderer
        .create(<Icon />)
        .toJSON()
      expect(tree).toMatchSnapshot()
    })
  })
})
