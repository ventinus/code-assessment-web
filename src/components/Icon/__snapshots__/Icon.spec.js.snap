// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icon component renders CartIcon correctly 1`] = `
<i
  className="icon icon--cart icon--15 "
/>
`;

exports[`Icon component renders EmptyCartIcon correctly 1`] = `
<i
  className="icon icon--cart icon--15 icon--empty-cart"
/>
`;
